tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generalize_zip_codes_test.py

```diff
@@ -140,7 +147,14 @@ def test_generalize_zip_codes_cleaning(self):
                  'StreetAddress_PIIZIP', 1585250, '723**'),
                 (1011, 1011, 0, datetime.strptime('2020-08-30',
                                                   '%Y-%m-%d').date(), 1011,
-                 'StreetAddress_PIIZIP', 1585247, '37218')
+                 'StreetAddress_PIIZIP', 1585247, '37218'),
+                (1012, 1012, 0, datetime.strptime('2020-08-30',
+                                                  '%Y-%m-%d').date(), 1012,
+                 'StreetAddress_PIIZIP', 1585250, '123*******'),
```

Comment:
@hiro-mishima, I think we also should make sure this will also show up as 123** instead of 123*******. so that all the zip codes are harmonized in the same format.